Added phone number for Pia

AlexRevsbech opened the pull request:
I have added Pia's phone number to the website.

AlexRevsbech committed:
Added phone number for Pia

revsbech commented on index.html:
```diff
@@ -37,7 +37,7 @@ <h3>Forår ferielukninger</h3>
 		  <h3>Sommerferie</h3>
 		  <p>Der er i år sommerferielukket i uge 27,28 og 29,   d.v.s. fra d. 30.6-18.7. , begge dage incl. </p>
-		  <p>Alle hverdage fra kl. 8-13 henvises til Læge Pia Kabrit og Læge Inger Højvig, Tlf: <a href="tel:45862655">4586 2655</a>
+		  <p>Alle hverdage fra kl. 8-13 henvises til Læge Pia Kabrit, Tlf: <a href="tel:45860140">4586 2655</a> og Læge Inger Højvig, Tlf: <a href="tel:45862655">4586 2655</a>
```
Are you sure this is the right phone-number?

AlexRevsbech replied:
I have now updated the code

revsbech reviewed (changes requested):
Looks good! 🚀  But I think you by mistake added the wrong number

AlexRevsbech committed:
Incorrect phone number and missing line

I have corrected Pia's phone number. Furthermore, the number was missing at another point in the code. Both have been corrected.

revsbech reviewed (approved):
Looks good. Let's merge it 🚀 🎸
